verification/helpers.go

```diff
@@ -32,6 +70,19 @@ func getArtifactPathFromUri(artifactUri string) (string, error) {
 	return artifactPath, nil
 }
 
+func getArtifactDigestFromUri(artifactUri string) (string, error) {
+	i := strings.LastIndex(artifactUri, ":")
+	if i < 0 {
+		return "", fmt.Errorf("artifact URI %q could not be parsed, make sure it is the fully qualified OCI artifact URI without the scheme/protocol. e.g domain.com:80/my/repository:digest", artifactUri)
```

Comment:
`digest` is digest like `sha256:xxx` or a tag like `v1`?

Reply:
Spec is not supporting tags yet. We can come back and update this code if we need to support tags.

Reply:
`domain.com:80/my/repository:digest` this reference string makes me confused. If it is for digest, it should be in the form of `domain.com:80/my/repository@digest`. That is, `@` instead of `:`.